<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>root_window</class>
 <widget class="QDialog" name="root_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>831</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="componate_alphabet_buttons"/>
  </widget>
  <widget class="QLabel" name="lbl_messege_show">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>311</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Закодированный текст ( в порядке следования квадратов)</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_messege_value">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Жёлудь и дерево красовались около ворот аббатсва проверка квантограббер Калининград Берляндский аэропорт десть часов пятнадцать минут день икс парад с флагами на площади близ фонтана</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="button_start">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>460</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>Старт игры</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>431</x>
     <y>13</y>
     <width>160</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="componate_set_show_n_m">
    <item>
     <layout class="QVBoxLayout" name="componate_n_m">
      <item>
       <layout class="QHBoxLayout" name="componate_n">
        <item>
         <widget class="QLabel" name="lbl_show_n">
          <property name="text">
           <string>n =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_n">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="componate_m">
        <item>
         <widget class="QLabel" name="lbl_show_m">
          <property name="text">
           <string>m =</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edit_m">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>5</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="componate_buttons">
      <item>
       <widget class="QPushButton" name="button_apply">
        <property name="text">
         <string>Подтвердить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_default">
        <property name="text">
         <string>По умолчанию</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
